templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -488,7 +451,10 @@
 
                                     <dp-date-range-picker
                                         class="u-2-of-4"
-                                        data-cy="procedureSettingsDate"
+                                        :data-cy="{
```

Comment:
Why this change? Doesn't `dataCy` have to be a string?

Reply:
see below